utils/sentry_client.py

```diff
@@ -0,0 +1,151 @@
+import requests
+import json
+
+class SentryClient:
+  ORGANIZATION = "sentry"
+
+  def __init__(self, host, token):
+    self.host = host
+    self.auth_token = token
+
+  def _do_sentry_api_call_(self, method, path, slugs, payload=None):
```

Comment:
Just makes it symmetrically clear that the function is intended for internal only.  I prefer the symmetric approach as it's more similar to other languages, but not sure if there is a python equivalent style

Reply:
The only case where a trailing underscore is reccomended is to avoid collisions with reserved keywords. And, in python, it looks off in any case, but that's due to the dunders, I guess.